tutorials/navigation/navigation_debug_tools.rst

```diff
@@ -1,56 +1,53 @@
 .. _doc_navigation_debug_tools:
 
-Navigation Debug Tools
+Navigation debug tools
 ======================
 
 .. note::
 
     The debug tools, properties and functions are only available in Godot debug builds. 
     Do not use any of them in code that will be part of a release build.
 
-Enabling debug navigation
+Enabling navigation debug
 -------------------------
 
-The navigation debug visualization is enabled by default inside the Editor.
-To visualize navigation meshes and connections also at runtime 
-enable the option ``Visible Navigation`` in the editor debug menu.
+The navigation debug visualizations are enabled by default inside the Editor.
+To visualize navigation meshes and connections at runtime too, enable the option ``Visible Navigation`` in the editor ``Debug`` menu.
 
 .. image:: img/navigation_debug_toggle.png
 
-In Godot debug builds the navigation debug can also be toggled on the NavigationServers from scripts.
+In Godot debug builds, the navigation debug can also be toggled through the NavigationServer singletons from scripts.
```

Comment:
```suggestion
In Godot debug builds the navigation debug can also be toggled through the NavigationServer singletons from scripts.
```
This break isn't helping and breaks the flow IMO, this is different from "If X, then Y"

Reply:
Fair enough. I removed the break.